pandas/core/computation/pytables.py

```diff
@@ -270,7 +271,7 @@ def evaluate(self):
             raise ValueError("query term is not valid [{slf}]".format(slf=self))
 
         rhs = self.conform(self.rhs)
-        values = [TermValue(v, v, self.kind).value for v in rhs]
+        values = list(rhs)
```

Comment:
this is the most non-trivial change.  It allows us to type the `kind` arg to `TermValue` as `str`, and in turn simplify some other parts of TermValue